src/main/java/com/moorhuhnservice/moorhuhnservice/data/Question.java

```diff
@@ -0,0 +1,34 @@
+package com.moorhuhnservice.moorhuhnservice.data;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.UUID;
+import javax.persistence.*;
+import lombok.AccessLevel;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.experimental.FieldDefaults;
+import org.hibernate.annotations.Type;
+
+@Entity
+@Data
+@NoArgsConstructor
+@FieldDefaults(level = AccessLevel.PRIVATE)
+public class Question {
+
+  @Id
+  @GeneratedValue(generator = "uuid")
+  UUID id;
+
+  String text;
+  String rightAnswer;
+
+  @ElementCollection
+  List<String> wrongAnswers;
```

Comment:
If possible:
```suggestion
  Set<String> wrongAnswers;
```
is far better for the performance of the database, and I don't know if we need the order of wrong answers...